packages/trader/src/Modules/Trading/Components/Form/screen-small.jsx

```diff
@@ -67,7 +69,16 @@ const CollapsibleTradeParams = ({
                     <RiskManagementInfo />
                 </div>
             )}
-            <div className='purchase-container'>
+            {is_turbos && (
+                <div>
+                    <PayoutPerPoint />
+                </div>
+            )}
+            <div
+                className={classNames('purchase-container', {
+                    'purchase-container__turbos': is_turbos,
```

Comment:
@hamza-deriv do we need to have this classname for turbos here? i can't find its declaration.
but let's try to handle styles for Purchase inside of the component using its internal classnames like we do it with multipliers and other contract types instead of adding this classname here 🙂

Reply:
This container class is used for mobile
![Screenshot 2023-02-08 at 8 36 43 AM](https://user-images.githubusercontent.com/120543468/217398819-51e758e7-f719-43e8-9205-ebea70b59418.png)


 `@include mobile {
        display: grid;
        grid-template-areas: 'a b';
        grid-column-gap: 0.5rem;

        &__turbos {
            display: block;
        }
    }`